cli/__snapshots__/cli_spec.js

```diff
@@ -16,8 +16,10 @@ exports['shows help for open --foo 1'] = `
   Opens Cypress in the interactive GUI.
 
   Options:
-    -b, --browser <browser-path>     path to a custom browser to be added to the
-                                     list of available browsers in Cypress
+    -b, --browser <browser-path>     runs Cypress in the browser with the given
+                                     name. if a filesystem path is supplied,
+                                     Cypress will attempt to use the browser at
```

Comment:
by will attempt does that mean it'll fail if it's not found or will fallback to a different browser?

Reply:
For `cypress run`, it will fail. For `cypress open`, it will show a message in the GUI and allow you to select a different browser.

Reply:
Sounds good. Might be something we can add/expand on in cypress-docs, but that'd be my first question as a user on how this would behave.